cmd/create/cluster/cmd.go

```diff
@@ -2611,6 +2629,93 @@ func run(cmd *cobra.Command, _ []string) {
 				os.Exit(1)
 			}
 
+			if interactive.Enabled() && !isGPUCountInputSet {
+				gpuLimitsInputCount, err = interactive.GetInt(interactive.Input{
+					Question: "Enter the number of GPULimitations you wish to set.",
+					Help:     cmd.Flags().Lookup(autoscalerGPULimitsCountFlag).Usage,
+					Default:  gpuLimitsInputCount,
+					Required: false,
+					Validators: []interactive.Validator{
+						nonNegativeIntValidator,
+					},
+				})
+				if err != nil {
+					r.Reporter.Errorf("Expected a valid value for %s: %s", autoscalerGPULimitsCountFlag, err)
+					os.Exit(1)
+				}
+				if err := intValidator(gpuLimitsInputCount); err != nil {
+					r.Reporter.Errorf("%s", err)
+					os.Exit(1)
+				}
+
+				GPULimitType := ""
+				GPULimits := []GPULimit{}
+				for GPULimitEntryIndex := 0; GPULimitEntryIndex < gpuLimitsInputCount && (GPULimitType != "" || GPULimitEntryIndex == 0); GPULimitEntryIndex++ {
+					GPULimitType = ""
+					GPULimitType, err = interactive.GetString(interactive.Input{
```

Comment:
The type+min+max seem like optional inputs, but I think in this case we should make them required